validator/validator.c

```diff
@@ -1635,14 +1645,15 @@ processInit(struct module_qstate* qstate, struct val_qstate* vq,
 	} else if(key_entry_isbad(vq->key_entry)) {
 		/* key is bad, chain is bad, reply is bogus */
 		errinf_dname(qstate, "key for validation", vq->key_entry->name);
-		errinf(qstate, "is marked as invalid");
+		errinf_ede(qstate, "is marked as invalid", LDNS_EDE_DNSSEC_BOGUS);
```

Comment:
This will cause a peculiar text message in the EDE I think. Do we need a `errinf_dname_ede()`?